.travis.yml

```diff
@@ -59,6 +59,7 @@ install:
   - conda config --env --add pinned_packages pyarrow==$PYARROW_VERSION
   - conda install -c conda-forge --yes codecov
   - conda install -c conda-forge --yes --file requirements-dev.txt
+  - pip install mlflow
```

Comment:
Out of curiosity, Is mlflow not in conda? If it;s uploaded, we can just write it into `requirements-dev.txt`.

Reply:
Actually mlflow is in conda, but seems like it breaks dependency with Python 3.5 for some reason.
https://travis-ci.com/databricks/koalas/jobs/201362227

Reply:
hm, then can we use conda and skip it for Python 3.5 one? We can do it in a followup too.

Reply:
I tried hard, but currently mlflow is not packaged for python 3.5 and 3.7. I suggest that until the 1.0 release, we keep it this way?

Reply:
FYI: mlflow for Python 3.7 is available on conda-forge.
If we keep this way, we need to add some note in `CONTRIBUTING.md` that we need to install mlflow separately for testing, especially for Python 3.5 users.

Reply:
Interesting, I was not aware we do put mlflow on conda. Maybe conda pulls it from Pip? I would definitely recommend loading it from pip for now.

Reply:
FYI: Here's the mlflow recipe. https://github.com/conda-forge/mlflow-feedstock

Reply:
(I don't think conda pulls it from PIP out of the box)